django_exportable_admin/utils.py

```diff
@@ -1,5 +1,6 @@
 import codecs
-import cStringIO
+from io import StringIO
```

Comment:
would you mind doing this as a try catch so cStringIO is still used if available?  Like
```
try:
  from cStringIO import StringIO
except ImportError:
  from io import StringIO
```
?